src/components/Tooltip/Tooltip.tsx

```diff
@@ -7,20 +7,30 @@ import Box from '../Box';
 
 const AnimatedTooltipPopup = animated(TooltipPopup);
 
+const DefaultWrapper: React.FC = ({ children }) => {
+  return (
+    <Box borderRadius="medium" bg="navyblue-300" p={4} m={2} fontSize="small" boxShadow="dark250">
+      {children}
+    </Box>
+  );
+};
+
 export interface TooltipProps {
-  /** The string or HTML that the tooltip will show*/
+  /** The string or HTML that the tooltip will show */
   content: string | React.ReactElement;
 
+  /** The wrapper that contains the contnet */
+  contentWrapper?: React.ElementType;
```

Comment:
optional: I'd rename that to just `wrapper`

Reply:
Done!